### ISSUE TITLE
Signature verification fails from locally signed transaction

### ISSUE THREAD

**Issue by abrkn** [ethereum/go-ethereum#1565]
Note: This following is done using 0.9.38-rc2 due to the inability to send transactions on live.

I'm working on a browser-based app where the signing work is based on the work from https://github.com/SilentCicero/ethereumjs-accounts/blob/master/index.js#L547

My "raw" transaction:

```
raw transaction: {
    "nonce": "00",
    "gasPrice": "0x00",
    "gasLimit": "0x00",
    "value": "400000000000000000",
    "data": "b214faa55550000000000000000000000000000000000000000000000000000000000000",
    "to": "0193d941b50d91be6567c7ee1c0fe7af498b4137"
}
```

serialized transaction: 

```
0xf88a800000940193d941b50d91be6567c7ee1c0fe7af498b413789400000000000000000a4b214faa555500000000000000000000000000000000000000000000000000000000000001ca0e7e2d8f03dc7b677f02afbe2c791115bbdf1280dc8b53a949efde1d6df4b84c8a0e7239306ff5e21af68c939d43cbc9805f71340d30cf2f1a5fee799d00447dd01
```

On my node I see:

```
I0731 19:29:25.238759       1 server.go:1775] http: panic serving 172.31.29.13:36091: runtime error: invalid memory address or nil pointer dereference
goroutine 4619 [running]:
net/http.func·011()
        /usr/lib/go/src/net/http/server.go:1130 +0xbb
math/big.(*Int).Cmp(0x0, 0xc20801eac0, 0xa94b00)
        /usr/lib/go/src/math/big/int.go:311 +0x24
github.com/ethereum/go-ethereum/crypto.ValidateSignatureValues(0x0, 0x0, 0x0, 0x8)
        /go-ethereum-0.9.38/build/_workspace/src/github.com/ethereum/go-ethereum/crypto/crypto.go:176 +0x3d
```

The signature values seem to indicate that something's gone terribly wrong. Ideas?

**Comment by fjl** [comment]
This is being fixed in #1541.

**Comment by abrkn** [comment]
Workaround: Set `gasPrice` and `gasLimit` to non-zero.
